prototypes/shared/src/avisi/apps/tech_testing_ground/prototypes/shared/core/items.clj

```diff
@@ -0,0 +1,9 @@
+(ns avisi.apps.tech-testing-ground.prototypes.shared.core.items)
+
+(def item-schema
+  [:map
+   {:decode/json #(into {} %)}
```

Comment:
Dit zou je kunnen vervangen door de functie `set`

Reply:
`set` results in a hash-set, I need to turn a java.util.Hashmap into a clojure-map. I believe this is the easiest way.

Reply:
On a second look I can actually remove this line here as the problem is now being solved elsewhere.